writer.go

```diff
@@ -26,8 +26,14 @@ import (
 	"sync"
 )
 
-// Discard is a convenience wrapper around ioutil.Discard.
-var Discard = AddSync(ioutil.Discard)
+var (
+	// Discard is a convenience wrapper around ioutil.Discard.
+	Discard = AddSync(ioutil.Discard)
+	// DiscardOutput is an Option that discards logger output.
+	DiscardOutput = Output(Discard)
+	// DiscardErrorOutput is an Option that discards logger error output.
```

Comment:
I think we should hold off on this till we see a lot of people asking for it, or we notice it in our own tests. `DiscardOutput` makes sense since we're already using it a lot in our tests.